terraform/deployments/cluster-services/aws_auth_configmap.tf

```
@@ -61,3 +75,47 @@ resource "kubernetes_cluster_role_binding" "cluster_admins" {
     api_group = "rbac.authorization.k8s.io"
   }
 }
+
+resource "kubernetes_cluster_role_binding" "cluster_readonly" {
+  metadata {
+    name = "cluster-readonly"
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "view"
+  }
+  subject {
+    kind      = "Group"
+    name      = "readonly"
+    api_group = "rbac.authorization.k8s.io"
+  }
+}
+
+resource "kubernetes_cluster_role" "read_crs_and_crbs" {
+  metadata {
+    name = "read_crs_and_crbs"
+  }
+
+  rule {
+    api_groups = ["rbac.authorization.k8s.io"]
+    resources  = ["clusterrolebindings", "clusterroles"]
+    verbs      = ["get", "list", "watch"]
+  }
+}
+
+resource "kubernetes_cluster_role_binding" "read_crs_and_crbs" {
+  metadata {
+    name = "read_crs_and_crbs"
```

(same suggestion as above, for convenience)
```suggestion
    name = "read-crs-and-crbs"
```

---

ok will do